uc_mcore/README.md

```diff
@@ -0,0 +1,5 @@
+# Under-constrained Symbolic Execution with Manticore
+
+```
+$ python uc_mcore.py -b uc_mcore -s crypto_scalarmult_curve25519_tweet_base -m example_model
```

Comment:
```suggestion
$ python uc_mcore.py --binary uc_mcore --symbol crypto_scalarmult_curve25519_tweet_base -m example_model
```

Reply:
I can't also find the `-m` flag in the argparse definition below, is this valid here?